Packs/EWS/Integrations/SecurityAndCompliance/SecurityAndCompliance.ps1

```diff
@@ -1478,6 +1498,8 @@ function Main {
     try {
         # Creating Compliance and search client
         $oauth2_client = [OAuth2DeviceCodeClient]::CreateClientFromIntegrationContext($insecure, $no_proxy)
+        $Demisto.Debug("oauth2_client from integration context: $($oauth2_client | ConvertTo-Json)")
```

Comment:
will it print the token? we should avoid doing that

Reply:
Issue debug log leftovers. Removing.